macosx/snes9x.xcodeproj/project.pbxproj

```diff
@@ -222,6 +222,7 @@
 		30D709C1236F7E3200AAB7C3 /* S9xPrefsViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 30D709BF236F7E3200AAB7C3 /* S9xPrefsViewController.m */; };
 		30D709C2236F7E3200AAB7C3 /* S9xPrefsViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 30D709C0236F7E3200AAB7C3 /* S9xPrefsViewController.xib */; };
 		30D709C5236F90DF00AAB7C3 /* S9xButtonConfigTextField.m in Sources */ = {isa = PBXBuildFile; fileRef = 30D709C4236F90DF00AAB7C3 /* S9xButtonConfigTextField.m */; };
+		600546B8250D160E00EC0977 /* .gitignore in Resources */ = {isa = PBXBuildFile; fileRef = 600546B7250D160D00EC0977 /* .gitignore */; };
```

Comment:
Looks like the .gitignore file got included in the snes9x.xcodeproj project. This could cause the .gitignore file to get copied into Snes9x.app's Resources directory on building.

To fix this, find the .gitignore file in the Xcode files sidebar, select it, hit the delete key, and choose to remove the reference, not the file in the dialog that appears.

Reply:
Added commit to remove reference.